tests/fatrace-currentmount

```diff
@@ -1,79 +0,0 @@
-#!/bin/sh
-set -euC
-
-LOG="$AUTOPKGTEST_TMP/fatrace.log"
-echo "starting fatrace..."
-fatrace --current-mount -s 2 -o $LOG &
-sleep 1
-
-echo "read a file..."
-head NEWS > /dev/null
```

Comment:
Replaced this with reading a file from the tmpdir mount, which makes more sense.